web/ui/react-app/src/components/SearchBar.tsx

```diff
@@ -4,17 +4,18 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
 
 export interface SearchBarProps {
-  handleChange: (e: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => void;
+  handleChange: (e: string) => void;
```

Comment:
Yeah, good idea in any case to only send a string to the parent, makes the interface cleaner :)

Reply:
thanks :)